File: app/src/main/java/de/sneak/sneakpeek/service/OMDBService.java
package de.sneak.sneakpeek.service;

import de.sneak.sneakpeek.data.Movie;
import retrofit2.http.GET;
import retrofit2.http.Query;
import rx.Observable;

public interface OMDBService {

    String BASE_URL = "http://www.omdbapi.com/";

    @GET("?&y=&plot=full&r=json")
    Observable<Movie> getMovie(@Query("t") String title);
}

File: app/src/main/java/de/sneakpeek/service/OMDBService.kt
package de.sneakpeek.service

import de.sneakpeek.data.Movie
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Query

interface OMDBService {

    @GET("?&y=&plot=full&r=json")
    fun getMovie(@Query("t") title: String): Observable<Movie>

    companion object {
        val BASE_URL = "http://www.omdbapi.com/"
    }
}
